### DISCUSSION TITLE
How to select and delete an empty child ?

### DISCUSSION THREAD

**Discussion by chrisvidal** [OFFLINE-GmbH/oc-boxes-support#176] [Help]
I inserted a child and didnt fill in its information (an image for that case)
so the child does not render in the editor and I can't select it to delete it.

is there a way other than coding in the htm partial for that sort of case and make a box of 50px x 50px ?

![image](https://github.com/user-attachments/assets/2496c714-da96-4896-8664-1430f6c76b8c)

**Comment by tobias-kuendig** [comment]
All boxes have special CSS classes and there is a global `.oc-boxes-edit-mode` class attached to the body when it is rendered in the Editor.

You can add a CSS rule that defines min height/width for any box.
